### ISSUE TITLE
a lot of equal answers in prediction, what to do?

### ISSUE THREAD

**Issue by lenkele** [dmlc/xgboost#1134]
The model is not bad, but it could be better. There a a lot of vectors (with close but different features) which get the same  predictions.
Can I do something to devide them?
parametre of accuracy of calculations? Or something else?

Thanks!

**Comment by khotilov** [comment]
I apologize for being blunt, but why would you be asking someone to basically teach you about ways to add more diversity to boosting models at a place that is for reporting bugs and keeping track of tasks and enhancements?

**Comment by lenkele** [comment]
There are many questions of this type here, in this tab called "issues". 
From what shoud I know that it is "place that is for reporting bugs "? 
Where it is better to ask about this?
 Will be grateful If you know the answer to the basic question...

**Comment by khotilov** [comment]
> There are many questions of this type here, in this tab called "issues".
 
And it's getting harder to go through the flood of stuff that should not belong here, when I have little time that I want to spend on checking the current issues of this project.
> From what shoud I know that it is "place that is for reporting bugs "? 

Readme at https://github.com/dmlc/xgboost
https://guides.github.com/features/issues/
> Where it is better to ask about this?

Again, see the readme. Also there's Stack Overflow, Cross Validated, Kaggle forums, etc. But be prepared to show  _your_ work in finding answers.
> Will be grateful If you know the answer to the basic question...

Trees divide vector space into discrete regions. Larger trees would create smaller regions. Subsampling parameters would help to add more diversity. Smaller learning rate would allow to scan the space more thoroughly. Plus, things may depend a lot on specifics of data.
